### ISSUE TITLE
Google APIs are not optional

### ISSUE THREAD

**Issue by Ldoppea** [bpetetot/conference-hall-beta#691]
**What is the issue?**

In the [Installation page](https://contribute-conference-hall.netlify.com/installation) Google APIs are declared `optional`, but they seems to be mandatory.

If Google APIs are not configured then we cannot create an event as the `Venue address` field is mandatory and Google APIs won't load for this field.

In this case the `Venue address` field accepts free text but the `mandatory` warning won't dismiss and so the `Create event` will stay grayed out.

**What is the expected behavior?**

Google APIs should be declared mandatory, or maybe the `Venue address` should be modified so he can work even when Google APIs are not configured.

**What is your environment?**

- OS: Windows 10
- Browser: Google Chrome 77.0.3865.120

**Comment by bpetetot** [comment]
Thanks for the feedback. Indeed now the venue address is a mandatory field when you create an event and we forgot to update the doc accordingly.

So we have 2 options:
1. Set google apis as mandatory in doc, but it can be quite annoying in dev mode
2. Venue address is a simple text field when we are in dev mode (with `process.env.NODE_ENV`) so he can work even when Google APIs are not configured.

**Comment by Ldoppea** [comment]
Second option seems better because Google API also ask to fill payment information. But the potential issue is that you won't be able to test this field with a production behavior.

What about a boolean env variable `REACT_APP_USE_MAP_API` ?

**Comment by bpetetot** [comment]
You're right but I prefer to use a  dedicated key for maps instead of reusing the firebase one:
`REACT_APP_GOOGLE_MAP_API_KEY`

if the key `REACT_APP_GOOGLE_MAP_API_KEY` is defined, enable Google Map API (everywhere) else disable GoogleMaps.

This should be mark as a breaking change in the application release note because we will not use the firebase key `REACT_APP_API_KEY`.
